libgcc/config/i386/cygming-crtend.c

```diff
@@ -58,7 +58,12 @@ extern void __gcc_deregister_frame (void);
 
 #pragma GCC diagnostic push
 #pragma GCC diagnostic ignored "-Wprio-ctor-dtor"
-static void register_frame_ctor (void); // TODO __attribute__ ((constructor (0)));
+#ifdef __aarch64__
+/* constructor not yet enabled for aarch64 */
+static void register_frame_ctor (void);
```

Comment:
I'd improve the code style, especially visually separate this code with empty lines like this:

```c
#pragma GCC diagnostic push
#pragma GCC diagnostic ignored "-Wprio-ctor-dtor"

#ifdef __aarch64__
// Constructor is not yet enabled for aarch64.
static void register_frame_ctor (void);
#else // __aarch64__
static void register_frame_ctor (void) __attribute__ ((constructor (0)));
#endif // !__aarch64__

#pragma GCC diagnostic pop
```

The same for the change bellow.